### ISSUE TITLE
How was Eddy planned to be addressed?

### ISSUE THREAD

**Issue by oesteban** [nipreps/EddyMotionCorrection#25]
I might be not seeing something, but SHOREline (https://github.com/mattcieslak/ohbm_shoreline/blob/master/cieslakOHBM2019.pdf) does not address Eddy...

**Comment by arokem** [comment]
Not mentioned explicitly but I think it should. Eddy currents from
diffusion encoding gradients are direction specific after all

**Comment by oesteban** [comment]
Well, the method is definitely sensitive to Eddy (as we saw in that example on our call a couple of weeks ago), but that doesn't mean that you can account for them.

If you only have one shell, then you'll converge to some central point of all the distributions of distorted orientations (each in a different way as you mention), but you don't inform the model with distortion-free data (unless you insert the B0 in the process).

I guess we will need to test this carefully. Let's get to the next milestone (the tutorial) first and we discuss what comes after in the following bi-weekly.

**Comment by dPys** [comment]
Yeah SHOREline is just the LOO prediction to correct for head motion, and it ideally assumes that the influence of eddy-currents have already been mostly controlled for. Registration of the dwis to B0's (i.e. which are not nearly as susceptible to the impact of eddy distortions as the rapidly-switching gradients of dwi) is really still the best method we have to correct for eddy since we can't quantify it directly. If there are eddy distortions present in the data that SHOREline trains on, I don't think we would expect the algorithm to make predictions that are eddy-free.

**Comment by dPys** [comment]
(though please correct me if I'm wrong on this!)

**Comment by arokem** [comment]
A bit more here. For empirical evaluation, it would be valuable to use data that has either (1) no eddy currents (e.g., twice-refocused spin echo (https://pubmed.ncbi.nlm.nih.gov/12509835/). There are possibly some datasets like that from the Stanford GE scanner I worked on as a postdoc, or (2) data where each gradient direction is acquired twice with reverse polarities. In the latter, we can attempt to compare with and w/o a separate correction for the eddy currents. @jelleveraart said he has access to a dataset that has an acquisition of this kind.

**Comment by dPys** [comment]
I'm in the process of generating simulated data right now in fiberfox, and doing this for a variety of acquisition schemes starting with various dipy datasets that can easily be fetched.  I also have some data already with the reverse-phase encoding. @arokem and @jelleveraart -- would you all be able to link me to the example with the twice-refocused spin echo?

**Comment by arokem** [comment]
Data I have access to is unfortunately already preprocessed: https://purl.stanford.edu/ng782rw8378, but we can ask Stanford folks if they could dig up some similar shareable data.
